Gemfile

```diff
@@ -5,14 +5,15 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem "activesupport", '~> 5.2.4.3'
 gem "ansible_tower_client", "~> 0.21.0"
-gem "cloudwatchlogger",   "~> 0.2.1"
+gem "cloudwatchlogger", "~> 0.2.1"
 gem "concurrent-ruby"
 gem "manageiq-loggers",   "~> 0.5.0"
 gem "manageiq-messaging", "~> 0.1.2"
 gem "more_core_extensions"
 gem "optimist"
 gem "prometheus_exporter", "~> 0.4.5"
 gem "rake", ">= 12.3.3"
+gem "rdkafka", "~> 0.8"
```

Comment:
Would we still need this if manageiq-messaging pulls it in? I'm guessing not, but also guessing this PR will include the `manageiq-messaging` branch changes also ( once that is ready. )

Reply:
Yep, exactly. I'll get it updated when the new branch is ready.